GitHub_Learning/Generic/AlertProvider/Application/AlertProviderController.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  AlertProvider.swift
+//  GitHub_Learning
+//
+//  Created by Amin Ashhab on 19/11/2020.
+//  Copyright © 2020 Amin_Second_Test_Project. All rights reserved.
+//
+
+import Foundation
+import CleanCore
+
+enum ActionStyle {
+    case _default
```

Comment:
Muzes pouzit primo default, kdyz ho zabalis do uvozovek. 
```suggestion
    case `default`
```